asset/quickstart/analyze-iam-policy-longrunning-bigquery/main.go

```diff
@@ -66,6 +66,16 @@ func main() {
 		log.Fatalf("AnalyzeIamPolicyLongrunning: %v", err)
 	}
 	fmt.Print(op.Metadata())
+
+	// Wait for the longrunning operation complete.
+	resp, err := op.Wait(ctx)
+	if err != nil && !op.Done() {
+		fmt.Println("failed to fetch operation status", err)
+	} else if err != nil && op.Done() {
+		fmt.Println("operation completed with error", err)
+	} else {
+		fmt.Println("operation completed successfully", resp)
```

Comment:
Is there something in the response we can print? We usually print a field or two, rather than the entire thing. Depends on the API.

Reply:
This response is an empty proto, all actual results are written into GCS or BQ, and useful metadata are in Operation.Metadata.